ServerCore/Pages/Events/FastestSolves.cshtml

```diff
@@ -15,19 +15,10 @@
 <h2>Fastest Solves</h2>
 <br/>
 <div>
-    <a class="radioButton"
-       asp-page="./FastestSolves"
+    <a asp-page="./FastestSolves"
        asp-route-sort="@Model.Sort"
-       asp-route-stateFilter="@FastestSolvesModel.PuzzleStateFilter.All">
-
-        <input id="StateFilter" name="StateFilter" value="All" type="radio" @(unsolvedFilter ? "" : "checked")> All
-    </a>
-    <a class="radioButton"
-       asp-page="./FastestSolves"
-       asp-route-sort="@Model.Sort"
-       asp-route-stateFilter="@FastestSolvesModel.PuzzleStateFilter.Unsolved">
-
-        <input id="StateFilter" name="StateFilter" value="Unsolved" type="radio" @(unsolvedFilter ? "checked" : "")> Unsolved
+       asp-route-stateFilter=@(unsolvedFilter ? FastestSolvesModel.PuzzleStateFilter.All : FastestSolvesModel.PuzzleStateFilter.Unsolved)>
```

Comment:
Another option:

I could change this code to something like:

```
static class PuzzleStateFilterMethods { 
    PuzzleStateFilter opposite(this PuzzleStateFilter psf) {
        switch (psf) {
            case All: 
                return Unsolved;
            case Unsolved:
            case default:
                return All;
        }
    }
}

FastestSolvedModel.PuzzleStateFilter oppositeFilter = Model.StateFilter.opposite();

asp-route-stateFilter=@(oppositeFilter)

Show @(nameof(oppositeFilter)) Puzzles
```

...or even add a uiString() extension method instead of just using nameof if we were worried about future compatability.

Advantages: looks cleaner
Disadvantages: if we ever add a third state we'd need to replace all this logic

Reply:
Nah no need